### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by Farida-Hamid**
## Steps:
- Add a column species of type string to the `animals` table.
- Insert data.
- ### Transactions:
    - **1**:
    Screen Shot 2022-10-25 at 6 52 32 PM

    - **2**:
   
    Screen Shot 2022-10-25 at 7 04 45 PM

    - **3**:
    
    Screen Shot 2022-10-25 at 7 13 28 PM

    - **4**:
    
    Screen Shot 2022-10-25 at 7 28 19 PM

- ### Queries:
    - **1**:
    
    Screen Shot 2022-10-25 at 8 02 43 PM

    - **2**:
    
    Screen Shot 2022-10-25 at 8 03 59 PM

    - **3**:
    
    Screen Shot 2022-10-25 at 8 04 51 PM
    
    - **4**:
    
    Screen Shot 2022-10-25 at 8 05 39 PM

    - **5**:

    Screen Shot 2022-10-25 at 8 06 28 PM

    - **6**:
    
    Screen Shot 2022-10-25 at 8 06 55 PM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -13,3 +13,40 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM
```
Comment (mikethreels) [inline queries.sql]: - [x] this query isn't finished yet.

File: queries.sql [hunk 2]
```diff
@@ -13,3 +13,40 @@ SELECT * FROM animals WHERE neutered = TRUE;
...
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM  
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
```
Comment (mikethreels) [inline queries.sql:28]: - [x] before the commit please also check if the changes are made.

File: queries.sql [hunk 3]
```diff
@@ -13,3 +13,40 @@ SELECT * FROM animals WHERE neutered = TRUE;
...
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```
Comment (mikethreels) [inline queries.sql:34]: - [x] before the rollback please also check run the SELECT query.

**Review by mikethreels — APPROVED** [review]
Hi Farida, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @Farida-Hamid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Project works properly 👍.
- use of gitflow 🚀.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**